lib/data/multipleBackendGateway.js

```diff
@@ -48,23 +115,52 @@ const multipleBackendGateway = {
     },
 
     get: (objectGetInfo, range, reqUids, callback) => {
+        const key = objectGetInfo.key ? objectGetInfo.key : objectGetInfo;
```

Comment:
If objectGetInfo.key exists the md-model-version is 2 or greater.
Otherwise, the objectGetInfo is just the key string.
That's from the data wrapper file :)